gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenComplexTypesSuite.scala

```diff
@@ -68,7 +68,7 @@ class GlutenComplexTypesSuite extends ComplexTypesSuite with GlutenSQLTestsTrait
         0.toFloat,
         0.toDouble,
         BigDecimal(0),
-        Array[Byte](0.toByte),
+        // Array[Byte](0.toByte),
```

Comment:
why remove？

Reply:
the ut is failed because of https://github.com/Kyligence/ClickHouse/issues/413 
it is fixed by another pr .